tests/rptest/tests/pkcs12_test.py

```diff
@@ -87,6 +88,7 @@ def _prepare_cluster(self, use_pkcs12: bool):
         self.rpk = RpkTool(self.redpanda, tls_cert=self.user_cert)
 
     @cluster(num_nodes=3)
+    @skip_fips_mode
```

Comment:
It looks like `use_pkcs12=False` is pretty much every other test, so no need to keep `False` in fips mode.